# Remote Management Request
temoto_core/RMPRequest rmp

# Rviz display plugin type
string type

# Rviz display plugin name
string name

# Topic that is used to send info to rviz plugin
string topic

# Plugin configuration XML string
string config

---

# Remote Management Response
temoto_core/RMPResponse rmp
